deploy-doc.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env sh
+# This is the script to deploy documentation only
+
+# abort on errors
+set -e
+
+# build
+npm run docs:build
+
+# navigate into the build output directory
+cd docs/.vuepress/dist
+
+git init
+git add -A
+git commit -m 'deploy'
+
+# push to gh-pages repo
+git push -f git@github.com:gtmills/phosphor-webui-vue.git master:gh-pages
```

Comment:
no way to do this without a force push? The force worries me.

Reply:
It shouldn’t matter since it is just the branch that has the static files. The development is in the docs dir and will be tracked with all other development files.